schedule/containers/sle_image_on_sle_host_podman.yaml

```diff
@@ -8,8 +8,8 @@ conditional_schedule:
       's390x':
         - installation/bootloader_start
   bci:
-    FLAVOR:
-      'Container-Image-Updates':
+    BCI_TESTS:
```

Comment:
is this in the variables.md? 
also double check that this is the right one and not `CONTAINERS_BCI_TESTS` as the commit msg describes.

Reply:
Good point. I will add it. VRs are on the way. Thanks.

Reply:
resolved. thanks